pypit/arflux.py

```diff
@@ -1,4 +1,7 @@
-# Module for fluxing routines
+""" Module for fluxing routines
+"""
+from __future__ import (print_function, absolute_import, division,
```

Comment:
(warning... rant beginning...)   I'm not so sure about the division import - but I'm open to it if needs be. In the past, I've been opposed to using it (and have considered it poor programming). Basically, it means that `1/2=0.5` instead of `1/2=0` (i.e. it converts two int values into a float). Sometimes, you may want to keep the integer division to return an integer. In any case, if just one of the numbers is real (and not integer), such as 1./2, you will get the desired result of 0.5. So, I've considered it to be somewhat lazy programming...  rant over...  :-)